docs/usage.md

```diff
@@ -34,58 +36,33 @@ manifest_json = json.dumps({
  })
 ```
 
-## Signing function
-
-The `sign_ps256` function is [defined in the library](https://github.com/contentauth/c2pa-python/blob/main/c2pa/c2pa_api/c2pa_api.py#L244) and  used in both file-based and stream-based methods. It's reproduced here to show how signing is performed.
-
-```py
-# Example of using Python crypto to sign data using openssl with Ps256
-from cryptography.hazmat.primitives import hashes, serialization
-from cryptography.hazmat.primitives.asymmetric import padding
-
-def sign_ps256(data: bytes, key: bytes) -> bytes:
-    private_key = serialization.load_pem_private_key(
-        key,
-        password=None,
-    )
-    signature = private_key.sign(
-        data,
-        padding.PSS(
-            mgf=padding.MGF1(hashes.SHA256()),
-            salt_length=padding.PSS.MAX_LENGTH
-        ),
-        hashes.SHA256()
-    )
-    return signature
-```
-
 ## File-based operation
 
 ### Read and validate C2PA data
 
-Use the `Reader` to read C2PA data from the specified asset file. 
+Use the `Reader` to read C2PA data from the specified asset file.
 
 This examines the specified media file for C2PA data and generates a report of any data it finds. If there are validation errors, the report includes a `validation_status` field.
 
-An asset file may contain many manifests in a manifest store. The most recent manifest is identified by the value of the `active_manifest` field in the manifests map. The manifests may contain binary resources such as thumbnails which can be retrieved with `resource_to_stream` or `resource_to_file` using the associated `identifier` field values and a `uri`.
+An asset file may contain many manifests in a manifest store. The most recent manifest is identified by the value of the `active_manifest` field in the manifests map. The manifests may contain binary resources such as thumbnails which can be retrieved with `resource_to_stream` using the associated `identifier` field values and a `uri`.
 
 NOTE: For a comprehensive reference to the JSON manifest structure, see the [Manifest store reference](https://opensource.contentauthenticity.org/docs/manifest/manifest-ref).
 
 ```py
 try:
-  # Create a reader from a file path
-  reader = c2pa.Reader.from_file("path/to/media_file.jpg")
-
-  # Print the JSON for a manifest. 
-  print("Manifest store:", reader.json())
-
-  # Get the active manifest.
-  manifest = reader.get_active_manifest()
-  if manifest != None:
-
-    # get the uri to the manifest's thumbnail and write it to a file
-    uri = manifest["thumbnail"]["identifier"]
-    reader.resource_to_file(uri, "thumbnail_v2.jpg") 
+    # Create a reader from a file path
+    with Reader("path/to/media_file.jpg") as reader:
```

Comment:
@crandmck I don't know if we need to emphasize that we recommend to use the context manager for proper memory clean up?

Reply:
Done, on top (after the import example).